client/src/use/useGirderDataset.ts

```diff
@@ -72,6 +72,7 @@ export default function useGirderDataset() {
       throw new Error(`could not fetch dataset for id ${datasetId}`);
     }
     dataset.value = folder;
+    return dataset.value.meta;
```

Comment:
It's kinda weird to load something then return a small part of it.  Maybe just return the whole thing.